tests/test_write.py

```diff
@@ -250,11 +250,15 @@ def test_write_data(mock_write_csv):
     KNOWN_WRITERS.clear()
     KNOWN_WRITERS.append(MagicMock(return_value=True))
     write_data(filename, data, comment, **csv_options)
-    KNOWN_WRITERS[0].assert_called_once_with(filename, data, comment, **csv_options)
+    KNOWN_WRITERS[0].assert_called_once_with(  # type: ignore [attr-defined]
+        filename, data, comment, **csv_options
+    )
     mock_write_csv.assert_not_called()
 
     KNOWN_WRITERS.clear()
     KNOWN_WRITERS.append(MagicMock(return_value=False))
     write_data(filename, data, comment, **csv_options)
-    KNOWN_WRITERS[0].assert_called_once_with(filename, data, comment, **csv_options)
+    KNOWN_WRITERS[0].assert_called_once_with(  # type: ignore [attr-defined]
```

Comment:
You could also get `mypy` to ignore the `tests` folder if you can't be bothered with all these `type: ignore`s.

Reply:
It has caught a few issues, so I rather not exclude the tests and just ignore the problematic lines.